charts/kubewarden-defaults/Chart.yaml

```diff
@@ -22,9 +22,9 @@ keywords:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.9.5-rc2
+version: 1.9.5-rc3
 # This is the version of Kubewarden stack
-appVersion: v1.12.0-rc2
+appVersion: v1.12.0-rc3
```

Comment:
If I got it right, that's the only chart where something changed. In that case the app version should not change